tests/unit/core/TestInstallAddon.js

```diff
@@ -1347,7 +1410,7 @@ describe(`${__filename}: withInstallHelpers`, () => {
       return uninstall(addon).then(() => {
         sinon.assert.calledWith(fakeTracking.sendEvent, {
           action: 'invalid',
-          category: UNINSTALL_CATEGORY,
+          category: getAddonEventCategory(addon.type, UNINSTALL_ACTION),
```

Comment:
You could declare a constant for `invalidType` at the top of the test and use it both to create the add-on and also here, instead of `addon.type`.